gh_deploy.sh

```diff
@@ -10,19 +10,7 @@ fi
 
 rev=$(git rev-parse --short HEAD)
 
-HSAMPLE_EVENT="${HSAMPLE_EVENT:-Open\ Tech\ Summit}"
```

Comment:
This was used previously for deploying Open Tech Summit 2016. Now, since we are already generating the Open Tech Summit 2017 Sample, there is no need of generating. Also, it is not mentioned in the list of sample events. So, I thought it would be better to remove it. Thanks!!